lib/ruby_llm/providers/bedrock/streaming.rb

```diff
@@ -1,10 +1,12 @@
 # frozen_string_literal: true
 
 require_relative 'streaming/base'
-require_relative 'streaming/content_extraction'
-require_relative 'streaming/message_processing'
-require_relative 'streaming/payload_processing'
-require_relative 'streaming/prelude_handling'
+require_relative 'streaming/content'
+require_relative 'streaming/messages'
+require_relative 'streaming/payload'
+require_relative 'streaming/prelude'
+require_relative 'streaming/tool_calls'
+require_relative 'tools'
```

Comment:
I know this was there before but we should try to not use `require_relative` as we're using zeitwerk.